test/gpu/native/studies/sort/radixSortGpu.chpl

```diff
@@ -516,52 +127,43 @@ module SortTest {
   proc main() {
     import Random;
     import Time;
-    if gpuDiags then
-      startGpuDiagnostics();
-    if verboseGpu then
-      startVerboseGpu();
+    import GpuSort;
+
+    if gpuDiags then startGpuDiagnostics();
+    if verboseGpu then startVerboseGpu();
 
     var timer: Time.stopwatch;
-    // var arr = [1,4, 2, 6, 3, 7, 5, 8] : uint;
     var arr: [0..<arrSize] uint;
     var seed = 17; // Make it deterministic
     Random.fillRandom(arr, seed);
-    if printArr then
-      writeln(arr);
-
-    timer.start();
-    if parallel then
-      RadixSort.parallelRadixSort(arr);
-    else
-      RadixSort.radixSort(arr);
-    timer.stop();
 
-    if printArr then
-      writeln(arr);
+    if printArr then writeln(arr);
 
-    if perftest then
-      writeln("Time: ", timer.elapsed());
-    if validate then
-      checkSorted(arr);
+    if parallel then on here.gpus[0]{
```

Comment:
nit: should this be named `useGpu` instead of `parallel`?

Reply:
The atomic sort still uses the GPU (although only for a part of the algorithm). I've renamed the config atomicSort and flipped the logic.